easybuild/easyconfigs/c/Commet/Commet-20160422-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'MakeCp' 
+
+name = 'Commet' 
+version = '20160422' 
```

Comment:
@boegel should it be 4ef0705 as shown on: https://github.com/pierrepeterlongo/commet/commits/master ?